nevergrad/benchmark/experiments.py

```diff
@@ -1,1841 +1,1856 @@
-# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Please run `black nevergrad` from nevergrad repository, hopefully that cleans this a bit... dunno what happened here

Reply:
It's due to plenty of ^M actually i guess.